.github/workflows/create-deployment-spack.yml

```diff
@@ -0,0 +1,46 @@
+name: Create Deployment Spack
+on:
+  workflow_dispatch:
+    inputs:
+      spack-version:
+        type: string
+        required: true
+        description: A version of spack
+      spack-packages-version:
+        type: string
+        required: true
+        default: main
+        description: A version of ACCESS-NRI/spack-packages
+      spack-config-version:
+        type: string
+        required: true
+        default: main
+        description: A version of ACCESS-NRI/spack-config
+jobs:
+  create-spack:
+    name: Spack
+    runs-on: ubuntu-latest
+    environment: "Gadi Spack"
+    steps:
+      - name: Setup SSH
+        id: ssh
+        uses: access-nri/actions/.github/actions/setup-ssh@main
+        with:
+          hosts: ${{ secrets.SSH_HOST }}
+          private-key: ${{ secrets.SSH_KEY }}
+      - name: Install
+        env:
+          SPACK_VERSION_LOCATION: ${{ vars.ROOT_SPACK_LOCATION }}/${{ inputs.spack-version }}
```

Comment:
Took me a while, but I found `vars.ROOT_SPACK_LOCATION` was set in the "Gadi Spack" environment. Would be good to add a comment with this information to make it obvious where to find it, and maybe change it if required.